src/samples/tester.cpp

```diff
@@ -0,0 +1,232 @@
+// Copyright 2017 Tunnler Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <deque>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+#include "tunnler/room.h"
+#include "tunnler/room_member.h"
+
+/**
+ * This will test the functions of the tunnler lib. 
+ * It will create a room listening on 127.0.0.1:1234 and two members connecting to that room.
+ * The output should look like:
```

Comment:
If you want to contain output in this document you should make clear that it's output. Indent it more or something